release-notes/topics/aap-21.adoc

```diff
@@ -0,0 +1,21 @@
+[[aap-2.1-intro]]
+= Ansible Automation Platform 2.1
+
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+
+.Enhancements
+
+* Added separate execution capacity and update your cluster without downtime with Automation Mesh that connects the cluster with a flexible communication method using receptor
```

Comment:
I recommend rewording this sentence to lead with Automation mesh. So something like "Automation mesh introduces a separated execution capacity ...."